src/streamlink/plugins/nimotv.py

```diff
@@ -46,21 +38,23 @@ class NimoTV(Plugin):
         250: '240p',
         500: '360p',
         1000: '480p',
-        2000: '720p',
+        2500: '720p',
         6000: '1080p',
     }
 
+    _re_appid = re.compile(br'appid=(\d+)')
+    _re_domain = re.compile(br'(https?:\/\/[A-Za-z]{2,3}.hls[A-Za-z\.\/]+)(?:V|&)')
+    _re_id = re.compile(br'id=([^|\\]+)')
+    _re_tp = re.compile(br'tp=(\d+)')
```

Comment:
These regexes should be good enough, even if it's not really possible to set proper anchors for extracting the data, as the anchors depend on the type of the preceding and following item in the JceStruct from the mStreamPkg data.

Since we're extracting strings from a JceStruct here, the two (?) bytes prior to the matching string value could be read and then decoded for the actual data type and length, so that the strings can be read precisely. That would make this approach a bit more proof without having to decode everything, but that's probably unnecessary and too complex.

Unless there's something wrong which I didn't notice, I'd say we should get this merged.